auth0/src/main/java/com/auth0/android/request/internal/RequestFactory.java

```diff
@@ -138,7 +147,9 @@ public AuthenticationRequest authenticationPOST(HttpUrl url, OkHttpClient client
     }
 
     AuthenticationRequest createAuthenticationRequest(HttpUrl url, OkHttpClient client, Gson gson, String method) {
-        return new BaseAuthenticationRequest(url, client, gson, method, Credentials.class);
+        BaseAuthenticationRequest request = new BaseAuthenticationRequest(url, client, gson, method, Credentials.class);
+        request.setJwtVerifier(verifier);
```

Comment:
If this is an "Authentication request", set the verifier. Internally, verification will only happen when an Id Token is present on the Credentials response.